### ISSUE TITLE
There's always a 'newer' version

### ISSUE THREAD

**Issue by j0hannes** [evnix/codowik#14]
The template page says `Update: New version 2017-06-20 is available.`

**Comment by topkecleon** [comment]
You can address this by opening `template.info.txt` and changing the `date` field to `2017-06-20`.

**Comment by j0hannes** [comment]
Thanks!

Shouldn't this be done best by the maintainer in the repo?

**Comment by topkecleon** [comment]
Yes, although I'm not sure if it would then be necessary to make the date current to reflect that the date was changed.
